solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -221,6 +238,7 @@ fi
 function setup_master_node(){
 echo "---------------------------------------[ Setting up Master Node $HOSTNAME ]--------------------------------------"
     download_deploy_script
+    download_log_script
```

Comment:
In `download_deploy_script` we are downloading all scripts so can we use the script from there itself?

Reply:
Done